CHANGELOG.md

```diff
@@ -1,6 +1,21 @@
 # Changelog
 All notable changes to the COBOL Language Support extension are documented in this file.
 
+# [0.18.0](https://github.com/eclipse/che-che4z-lsp-for-cobol/compare/0.17.1...0.18.0) (2021-03-12)
+
+### Bug Fixes
+* Add color to hover message ([f15e867](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/f15e867c0d3af03f75012b40ca6c83360e04f35e))
+* Add coloring for divisions ([988fd45](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/988fd457fe447b7c84846301e423648485314fca)), closes [#796](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/796)
+* Fix misspelled Keyword Warning Shown on a Correct Token [#760](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/760) ([8958921](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/8958921e2073847ab1cccb344556e491cf502c34))
+
+### Features
+* Add COBOL world pattern to IDE editor ([37213a7](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/37213a7ae6dd26b8367e3f5cc4a26d9bf3392928)), closes [#798](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/798)
```

Comment:
Fix autocomplete for identifiers separated with dashes

Reply:
What is this a comment on? Is it an extra PR which didnt make it into the autogenerated list?

Reply:
It is a description of this feature from the user point of view

Reply:
OK done